samples/sensor/stm32_vbat_sensor/sample.yaml

```diff
@@ -0,0 +1,16 @@
+sample:
+  description: Usage of STM32 Vbat internal sensor
+  name: stm32_vbat_sensor
+tests:
+  sample.sensor.stm32_vbat_sensor:
+    depends_on: adc
+    tags: sensors
+    platform_allow: nucleo_g071rb
+    timeout: 10
+    harness: console
+    harness_config:
+      type: multi_line
+      ordered: true
+      regex:
+        - "STM32 Vbat sensor test"
+        - "Current Vbat voltage: 33(.*) mV"
```

Comment:
Has to be updated to Volts.

Reply:
changed to` "Current Vbat voltage: 3(.*) V"`